spec/resources/sites_spec.rb

```diff
@@ -3,13 +3,36 @@
 describe TableauApi::Resources::Sites, vcr: { cassette_name: 'sites' } do
   include_context 'tableau client'
 
+  describe '#create' do
+    let(:default_client) do
+      TableauApi.new(
+        host: ENV['TABLEAU_HOST'],
+        site_name: 'Default',
+        username: ENV['TABLEAU_ADMIN_USERNAME'],
+        password: ENV['TABLEAU_ADMIN_PASSWORD']
+      )
+    end
+
+    it 'can create a site' do
```

Comment:
Is it possible to add a failure? Maybe by creating a site with a duplicate name?